onnxruntime/core/providers/cpu/tensor/upsamplebase.h

```diff
@@ -237,6 +238,9 @@ class UpsampleBase {
     if (coordinate_transform_mode_name == "half_pixel") {
       return HALF_PIXEL;
     }
+    if ("half_pixel_symmetric" == coordinate_transform_mode_name) {
```

Comment:
Nit. Can we also change the previous "==" comparisons to use the same pattern?

Reply:
Why is a different pattern needed? We build with warnings enabled and I believe those would catch any accidental assignment that reversing the arguments attempts to avoid.

Reply:
not being consistent with existing style (what I did) is not good. according to @skottmckay , I make the change so that it is consistent with existing code.